.github/workflows/examples.yml

```diff
@@ -206,20 +159,14 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: 'Free up disk space'
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
+
+    - name: Install cypress deps
```

Comment:
Should this be put into a separate commit/PR ? I'd say that some tests were failing after this addition.

Reply:
Yes, I added another commit to try to get this green for integration.